js/dialogcards.js

```diff
@@ -476,11 +674,45 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
       self.stopAudio(self.$current.index());
       self.$current.removeClass('h5p-dialogcards-current');
       self.$current = $prev.addClass('h5p-dialogcards-current').removeClass('h5p-dialogcards-previous');
+      self.turnCardToFront ();
       self.updateNavigation();
     }
   };
 
   /**
+   * User selected cards order option (normal/random).
+   */
+  C.prototype.randomizeOrder = function (cardsOrder) {
+    var self = this;
+    self.randomCards = cardsOrder;
+    var el = '.h5p-dialogcards-order';
+    $( el ).remove();
```

Comment:
Well, it does work. However, how would I search in the current instance of DC only?